.circleci/config.yml

```diff
@@ -44,6 +53,12 @@ jobs:
       - store_artifacts:
           path: ./docs/build/html
 
+      - save_cache:
+          key: tutorial-{{ checksum "/tmp/tutorial_files.md5" }}
+          paths:
+            - tutorial/MNIST
```

Comment:
Sorry but let me ask a question because I'm not familiar with the original PR and CircleCI: when `tutorial/MNIST` will be generated? I checked it appear in the original [PR](https://github.com/optuna/optuna/pull/1722/files#diff-e4697a2845e32e857d02b8540419a37459fa603b4db72167817367038b4f5a64R30) but I couldn't what step creates it...

Reply:
The visualization tutorial with PyTorch of #1722 downloaded MNIST. Then, it turned out to be not feasible on ReadTheDocs due to the longer build time and we migrated to lightgbm from PyTorch in #2088, which might be tentative though. So now, we don't need MNIST at this moment.